specs/binding-http.spec/src/main/scripts/io/aklivity/zilla/specs/binding/http/schema/http.schema.patch.json

```diff
@@ -388,6 +388,548 @@
+                                                                ]
+                                                            },
+                                                            {
+                                                                "required":
+                                                                [
+                                                                    "headers"
+                                                                ]
+                                                            }
+                                                        ],
+                                                        "additionalProperties": false
+                                                    }
+                                                }
+                                            },
+                                            "anyOf":
+                                            [
+                                                {
+                                                    "required":
+                                                    [
+                                                        "path",
+                                                        "headers"
+                                                    ]
+                                                },
+                                                {
+                                                    "required":
+                                                    [
+                                                        "path",
+                                                        "params"
+                                                    ]
+                                                },
+                                                {
+                                                    "required":
+                                                    [
+                                                        "path",
+                                                        "content"
+                                                    ]
+                                                },
+                                                {
+                                                    "required":
+                                                    [
+                                                        "path",
+                                                        "responses"
+                                                    ]
+                                                }
+                                            ],
+                                            "additionalProperties": false
+                                        }
+                                    }
+                                },
+                                "additionalProperties": false
+                            }
+                        }
+                    }
```

Comment:
Given that `server` options are a superset of `client` options, is there not a way to specify the common subset once, and then conditionally include the additional options properties for `server`?